### ISSUE TITLE
Move Google Analytics tracker id from app.module.ts to config.service

### ISSUE THREAD

**Issue by bytech-ca** [eclectic-bytech/webcourses-ngx-v4#148]
Move tracker code from `/app.module.ts` to `/core/services/config/config.service`.

**Comment by bytech-ca** [comment]
`ConfigService` loads the `/resources/webcourses/src/config.json` file (when running compiler in watch mode) and `/public/webcourses/config.json` (when running on homestead.

ConfigService is implemented in several areas. Grepping for it will show where and how its implemented.

**Comment by bytech-ca** [comment]
Since it looks like the config file loads too late for this purpose, let's go with configuring the environment variables: https://www.digitalocean.com/community/tutorials/angular-environment-variables

**Comment by bytech-ca** [comment]
Please

- [ ] Merge master into this branch to make sure there are no conflicts which might be best resolved by branch's author
- [ ] Do a Pull Request to signal to the PM that this branch is up to date and ready for review

**Comment by mrjamiecakes** [comment]
Pending pull request by @SirL3X0N .

**Comment by SirL3X0N** [comment]
@mrjamiecakes No it has been completed in under 30 minutes, It has been sitting in pull requests for four days waiting on @bytech-ca

**Comment by mrjamiecakes** [comment]
> g in pull requests for four days waiting on

Yeah thats what I was saying. I tagged him to let him know its being is our pull que.
